src/processes.c

```diff
@@ -211,12 +214,17 @@ typedef struct procstat
 	derive_t io_syscr;
 	derive_t io_syscw;
 
+	derive_t cswitch_vol;
+	derive_t cswitch_invol;
+
 	struct procstat   *next;
 	struct procstat_entry_s *instances;
 } procstat_t;
 
 static procstat_t *list_head_g = NULL;
 
+static _Bool ps_ctx_switch = 0;
```

Comment:
The naming convention for variables like this is `report_foo`, i.e. `report_ctx_switch` here.